### DISCUSSION TITLE
Slack/Discord Server?

### DISCUSSION THREAD

**Discussion by adampankow** [Security-Onion-Solutions/securityonion#3387] [General] [answered]
Has any consideration been given to standing up a Slack (or Discord) Server for the Security Onion project? I find Slack and Discord servers to be a great real-time way for users in the community to ping questions and/or ideas off of each other, in addition to project personnel weighing in on issues.

**Comment by dougburks** [comment, marked as answer]
We considered Slack/Discord but decided that Github Discussions is a better way to organize discussions and make them searchable by search engines.

**Reply by adampankow** [reply]
I could understand that thought process. As always, thanks for the reply and insight, Doug!

Hope all has been well for you and yours, and awesome to see the business side of things continuing to grow!
